config/coreboot-nitropad-ns50.config

```diff
@@ -0,0 +1,44 @@
+CONFIG_LOCALVERSION="v1.6.0"
```

Comment:
Please use coreboot helper equivalent of linux module to save in oldconfig format. Same for other board if course

Reply:
done